simple_app/templates/solo_result.html

```diff
@@ -16,6 +16,13 @@ <h2>Names</h2>
     <div class="block">
         <h2>Photometry</h2>
         <p style="line-height:1.5em; font-size: x-large">{{ everything.photometry|safe }}</p>
+        <div class="block" id="camdplot">
```

Comment:
An alternative to embedding the plot is using something like {{ plot|safe }} which requires something like `script=script, plot=plot` to be passed to `render_template` in the python code which in turn come from something like `script, plot = components(bokeh_plot)`. (See example at https://github.com/dr-rodriguez/AstrodbWeb/blob/2e1210822fb85c47ff6db286843a0ff4b829f5b7/bdnyc_app/app.py#L445 though that is a bit old now)
However, your javascript approach could be more flexible so this is also OK.

Reply:
Sure, I had considered that; it's functionally identical (embedding a jsonified bokeh object); either way should work including any javascript actions